Source/CrossingDetector/CrossingDetector.cpp

```diff
@@ -61,6 +60,8 @@ CrossingDetector::CrossingDetector()
     jumpSize.resize(pastSpan + futureSpan + 2);
```

Comment:
We need to change the array resizing for multiple instances

Reply:
OK yup, I can change all of those.